tflib/publisher/main.tf

```diff
@@ -24,12 +24,33 @@ variable "extra_packages" {
   default = ["wolfi-baselayout"]
 }
 
+variable "name" {
+  type    = string
+  default = "TODO" // TODO remove this
+}
+
+output "path" {
+  value = basename(path.cwd)
+}
+
+locals {
+  updated_config = merge(yamldecode(var.config),
+    var.name == "TODO" ? {} :
```

Comment:
what isn't migrated, or is this for merge conflicts?

Reply:
This is mainly here for in-flight PRs that do it the old way, so we can clean those up in a post-merge pass.